site/src/pages/release-notes/1.0.0.mdx

```diff
@@ -0,0 +1,211 @@
+---
+date: 2020-08-21
+---
+
+## 🌟 New features
+
+- You can now check whether a <type://RequestContext> contains an attribute or not conveniently using
+  <type://RequestContext#hasAttr(AttributeKey)> and <type://RequestContext#hasOwnAttr(AttributeKey)>. #3014
+- You can now specify the default service and log name programmatically for an annotated service. #2979
+  - It was previously only possible to specify a service name with <type://@ServiceName> annotation.
+  ```java
+  Server.builder()
+        .annotatedService().defaultServiceName("my-service")
+                           .build(myAnnotatedService);
+  ```
+- Annotated services now support [Kotlin coroutines](https://kotlinlang.org/docs/reference/coroutines-overview.html).
+  #2983 #3010 #3011
+  - See [Kotlin coroutines support](/docs/server-annotated-service#kotlin-coroutines-support) for more
+    information.
+  ```kotlin
+  class MyAnnotatedService {
+      @Get("/users/{name}")
+      suspend fun getUserToken(@Param name: String): String {
+          val user = myReactiveRepository.findByName(name).await()
+          return user.token
+      }
+  }
+  ```
+- You can now get the current <type://ServiceRequestContext> from the Kotlin coroutine context when using
+  [grpc-kotlin](https://github.com/grpc/grpc-kotlin). #3001
+- You can now use <type://GrpcMeterIdPrefixFunction> to create meters with a `grpc-status` tag for gRPC clients
+  and servers. #2762 #2971
+  ```java
+  Server.builder()
+        .route().defaultServiceName("my-grpc-services")
+                .build(GrpcService.builder()
+                                  .addService(serviceFoo)
+                                  .addService(serviceBar)
+                                  .build()
+                                  .decorate(MetricCollectingService.newDecorator(
+                                      GrpcMeterIdPrefixFunction.of('my.grpc.svcs'))))
```

Comment:
nit: need to use `"`

Reply:
Oops! Too much JS/TS recently :sweat_smile: